src/modules.d.ts

```diff
@@ -109,3 +109,9 @@ declare module 'electron' {
         export function readText(type?: string): string;
     }
 }
+
+declare namespace Rx {
```

Comment:
Why do we need this if you added Rx's types in `package.json`?

Reply:
Because scheduleFuture function is missing in rx type definition.  This it the patch.